### PR TITLE
Nawqaan 405

### PR DESCRIPTION
**Pull request by mbucknell**

### CODE REVIEW

File: nar_common/templates/nar_ui/full_report.html [hunk 1]
```diff
@@ -94,6 +94,13 @@
 
 	<div class="report_filter col-lg-3 col-md-3 col-sm-3 col-xs-3">
 		<div id="plotToggleTree"></div>
+		<div id="plotHelpDiv">
+		  <ul>
+		      <li id="sampleConcentrations"><span class="plotHelpTerm"></span><a href="#' data-toggle="popover">?</a></li>
```
Comment (jiwalker-usgs) [inline nar_common/templates/nar_ui/full_report.html]: Looks like you have mismatched quotes here ``<a href="#'``

Comment (mbucknell) [inline nar_common/templates/nar_ui/full_report.html]: I knew there was something here but couldn't find it. Thanks!

**Comment by jiwalker-usgs** [comment]
The sorting logic is a bit complex as it sits.  It is worth thinking about moving the comparison logic to a comparator to simplify the logic within the .sort() itself.

**Comment by mbucknell** [comment]
I could add two functions, one for branch sorting and one for leaf sorting.

**Comment by jiwalker-usgs** [comment]
Yeah, I'm thinking something along the lines of ``nodeA.compare(nodeB)`` where the logic of comparison and ordering can be moved elsewhere.

**Comment by mbucknell** [comment]
If we had a defined "node" object, I think this would be worth it, but I don't think it is in this situation unless we did more major refactoring. Probably not worth it at the moment.

**Comment by jiwalker-usgs** [comment]
Yeah, I agree, more of a thought exercise I guess.

**Pull request merged**